runner/logs.go

```diff
@@ -16,12 +16,12 @@ import (
 	"github.com/pkg/errors"
 )
 
-func downloadLogsForServer(server state.Server, globalCollectionOpts state.CollectionOpts, logger *util.Logger) (state.PersistedLogState, bool, error) {
```

Comment:
I believe the `success` `bool` had the same code paths as `error == nil` so I don't think we need it.

Reply:
Nope never mind, just caught where this was different. Restoring it.